packages/backup-restore-operator/charts/templates/serviceaccount.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "backupRestore.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "backupRestore.labels" . | nindent 4 }}
```

nit, `nindent` should automatically indent the first line iirc. Happens a couple of times throughout the PR.

```suggestion
  labels: {{- include "backupRestore.labels" . | nindent 4 }}
```

---

yeah I was using the sample chart helm3 creates and it added the labels in this format (next line) so thought I'd let it be 😅